cli/explain/compactgraph/compactgraph_test.go

```diff
@@ -835,7 +835,7 @@ func TestChainOfLocalChanges(t *testing.T) {
 }
 
 func diffLogsAllowingError(t *testing.T, name, bazelVersion string) ([]*spawn_diff.SpawnDiff, error) {
-	dir := "buildbuddy/cli/explain/compactgraph/testdata"
```

Comment:
nit: we can probably use `x_defs` + `rlocationpath` here to avoid referencing the repo name

Reply:
That would require `filepath.Dir` logic though since we reference many different files under a common directory. Since it's just one location, I lean towards leaving this as is.